svgdigitizer/entrypoint.py

```diff
@@ -58,6 +58,12 @@ def cli():
     help="Detect non-orthogonal skewed axes going through the markers instead of assuming that axes are perfectly horizontal and vertical.",
 )
 
+si_option = click.option(
+    "--si",
+    is_flag=True,
+    help="Converts units of the plot and CSV to SI (if possible).",
```

Comment:
```suggestion
    help="Convert units of the plot and CSV to SI (if possible.)",
```

Also, when would it not be possible?